api/src/utils/errors.ts

```diff
@@ -0,0 +1,9 @@
+export type AppError = {
```

Comment:
Is this needed? maybe the standard Error constructor take care of this and more https://nodejs.org/api/errors.html#errors_new_error_message

Reply:
We'll come back to this refactor if needed